samples/apps/copilot-chat-app/webapi/Program.cs

```diff
@@ -30,10 +34,20 @@ public static async Task Main(string[] args)
             .AddSemanticKernelServices()
             .AddPersistentChatStore();
 
-        // Add in the rest of the services.
+        // Add AppInsights telemetry
         builder.Services
-            .AddApplicationInsightsTelemetry()
+            .AddHttpContextAccessor()
+            .AddApplicationInsightsTelemetry(options => { options.ConnectionString = builder.Configuration["APPLICATIONINSIGHTS_CONNECTION_STRING"]; })
```

Comment:
This may be considered a breaking change. This is the standard connection string used across Azure in environment variables etc. (https://learn.microsoft.com/en-us/azure/azure-monitor/app/sdk-connection-string?tabs=dotnet5)